client/src/app/barmen/edit/barman-edit.component.ts

```diff
@@ -49,7 +49,8 @@ export class BarmanEditComponent implements OnInit {
             lastName: new FormControl('', [Validators.required]),
             firstName: new FormControl('', [Validators.required]),
             nickname: new FormControl('', [Validators.required]),
-            facebook: new FormControl(''),
+            facebook: new FormControl('',
+                [Validators.pattern(/^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/)]),
```

Comment:
We should even check for a facebook link! (starting with`https://www.facebook.com/`)